pyp5js/fs.py

```diff
@@ -79,6 +80,14 @@ def target_dir(self):
     def __eq__(self, other):
         return self.sketch_name == other.sketch_name
 
+    @property
+    def urls(self):
+        SketchUrls = namedtuple('SketchUrls', ['p5_js_url', 'sketch_js_url'])
```

Comment:
Why not share the namedtuple to the whole module (put its declaration outside the class)? The way it is, this property will create objects from different classes all the time:

```python
>>> namedtuple("A", ["a", "b", "c"]) is namedtuple("A", ["a", "b", "c"])                                          
False
```